content/01.abstract.md

```diff
@@ -1,27 +1,3 @@
 ## Abstract {.page_break_before}
 
-Manubot is an open source tool for writing manuscripts on GitHub in markdown format.
-Manubot applies the git-based software workflow to scholarly writing, enabling enhanced transparency, collaboration, automation, and reproducibility.
-
-This manuscript is a Manubot demo, intended to give users a playground to practice using Manubot.
-Everyone is encouraged to try writing with Manubot by editing this manuscript.
-
-Manubot is described in the paper titled "Open collaborative writing with Manubot" [@url:https://greenelab.github.io/meta-review].
```

Comment:
Some of this we should keep because it documents the purpose of the manuscript. The rest we can delete.